scripts/bootstrap

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${ECHOLOCATOR_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "${0}")/../"
+
+function usage() {
+    echo -n \
+        "Usage: $(basename "$0")
+Pull down secrets from S3.
+"
+}
+
+function pull_env() {
+    pushd "${DIR}"
+
+    echo "Pulling .env from s3://${1}"
+    # aws s3 cp "s3://${1}/.env" ".env"
```

Comment:
I am assuming that we would pull some env vars from S3 here. Please let me know if this should be something else.

Reply:
Ah, this is a very good point. I agree- it does not make much sense to me either to copy the `env.yml`from the template file and populate the values, since:1. we would potentially move away from `mastarm`; 2. the values would need some human op to grab from last pass and fill in manually. And yea, we indeed do not need any env vars for the Django dev yet. I am with you on leaving this `bootstrap` script as is and treat the current code comment as a placeholder!